test/preprocessor.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
+const babel = require('babel-core');
+
+module.exports = {
+  process(src) {
+    return babel.transform(
+      src,
+      JSON.parse(fs.readFileSync('./examples/.babelrc')),
```

Comment:
what is this used for?

Reply:
or why do we need to do this specifically vs having one babelrc

Reply:
We can't have one babelrc. Rollup needs to have module: false so that it can take over.
Tests and examples need modules: true since they are not bundled.

Reply:
i think we have too many tho, I really want to avoid a lot of babel configs as they very easily get out of sync and cause annoying bugs. I would put the root `.babelrc`back, and in the rollup script require it (it's json) and tweak the modules setting and pass the config in to the plugin directly. That or use an env var and the babel `env` key to define a different one in the same file for building

Reply:
It quite common to have two babel configs. Can we keep it that way? The tweaks you suggest add complexity. Not forgetting the loss of declarativeness :(

Reply:
having three configs adds a lot more complexity and chances for error than one and a tweak to it. It doesn't make sense that the build script uses the example/dev scratch pad babel config either...

i still think that if you declarativeness the simplest way to do that is a single babelrc that specifies different `env` branches